packages/portal/src/components/search/SearchInterface.vue

```diff
@@ -156,16 +156,23 @@
           v-if="advancedSearchEnabled"
           class="d-flex justify-content-between align-items-center flex-nowrap"
         >
-          <b-button
-            aria-controls="search-query-builder search-query-builder-mobile"
-            :aria-expanded="showAdvancedSearch"
-            class="search-toggle query-builder-toggle m-3"
-            :class="{ 'open': showAdvancedSearch }"
-            variant="link"
-            @click="toggleAdvancedSearch"
-          >
-            {{ $t('search.advanced.show', { 'showOrHide': showAdvancedSearch ? $t('actions.hide') : $t('actions.show') }) }} {{ advancedSearchQueryCount ? `(${advancedSearchQueryCount})` : '' }}
-          </b-button>
+          <span>
+            <b-button
+              aria-controls="search-query-builder search-query-builder-mobile"
+              :aria-expanded="showAdvancedSearch"
+              class="search-toggle query-builder-toggle ml-3 my-3"
+              :class="{ 'open': showAdvancedSearch }"
+              variant="link"
+              @click="toggleAdvancedSearch"
+            >
+              {{ $t('search.advanced.show', { 'showOrHide': showAdvancedSearch ? $t('actions.hide') : $t('actions.show') }) }} {{ advancedSearchQueryCount ? `(${advancedSearchQueryCount})` : '' }}            </b-button>
```

Comment:
this is missing a line break looks like.

Reply:
Added a line break.

Reply:
Added a line break.